scio-smb/src/main/java/org/apache/beam/sdk/extensions/smb/SortedBucketIO.java

```diff
@@ -110,12 +128,14 @@ private CoGbkWithSecondaryBuilder(Class<K1> primaryKeyClass, Class<K2> secondary
      * Returns a new {@link CoGbkWithSecondary} with the given first sorted-bucket source in {@link
      * Read}.
      */
-    public CoGbkWithSecondary<K1, K2> of(Read<?> read) {
```

Comment:
Looks like this method got deleted?

Reply:
My IDE made a lot of white space changes, I made a bunch of mistakes fixing them back :)